bin/crew

```diff
@@ -616,11 +599,7 @@ def prop(silent = false)
 end
 
 def whatprovides(regexPat)
-  @crew_manifest_search_dir = ENV['CREW_TESTING'] == '1' ? "#{CREW_MANIFEST_CACHE_DIR}/chromebrew-#{CREW_TESTING_BRANCH}/manifest/#{ARCH}" : "#{CREW_MANIFEST_CACHE_DIR}/chromebrew-#{CREW_GITHUB_BRANCH}/manifest/#{ARCH}"
-
-  updatefiles unless Dir.exist?(@crew_manifest_search_dir) && !Dir.empty?(@crew_manifest_search_dir)
-
-  matchedList = `grep -r "#{regexPat}" #{@crew_manifest_search_dir}`.lines(chomp: true).flat_map do |result|
+  matchedList = `grep -R "#{regexPat}" #{CREW_LIB_PATH}/manifest/#{ARCH}`.lines(chomp: true).flat_map do |result|
```

Comment:
Add?
```
userspace_arch = ARCH.include('aarch64') ? 'armv7l' : ARCH
matchedList = `grep -R "#{regexPat}" #{CREW_LIB_PATH}/manifest/#{userspace_arch}`.lines(chomp: true).flat_map do |result|
```
Might make sense to add a new value in constant.rb too.

Reply:
See https://github.com/chromebrew/chromebrew/pull/8425.

Reply:
Change made